test/log.ts

```diff
@@ -15,19 +15,19 @@
  */
 
 import {util} from '@google-cloud/common-grpc';
-import * as promisify from '@google-cloud/promisify';
+import * as callbackify from '@google-cloud/promisify';
 import * as assert from 'assert';
 import * as extend from 'extend';
 import * as proxyquire from 'proxyquire';
 
-let promisifed = false;
-const fakePromisify = extend({}, promisify, {
-  promisifyAll(c, options) {
+let callbackified = false;
+const fakecallbackify = extend({}, callbackify, {
```

Comment:
`fakeCallbackify` to keep the camelcase style.

Reply:
Nice catch 👍 
Done